docs/getting-started/overview.rst

```diff
@@ -1,6 +1,113 @@
-.. include:: /stub-topic.txt
+Overview of .NET implementations
+================================
+
+.NET Platform is, at its very core, a set of standards that can be implemented. There are 
+various implementations, some coming from Microsoft, some coming from other companies and 
+groups. This document will outline the current state of the .NET ecosystem and its various 
+implementations, so you can get a good "lay of the land". 
+
+.NET Core
+---------
+.NET Core is a cloud-optimized, cross-platform implementation of the .NET Platform. It currently 
+supports three main operating systems: Linux, Windows and OS X. 
+
+There are several reasons why we built .NET Core.
+
+**Cross-platform support** is a requirement of many developers today. It's important to use the 
+platform that has the characteristics required by the application, which, in turn, can run on the 
+platform required by users or perhaps is prescribed by corporate policy.
+
+**Open Source** because it has proven to be the best possible way to enable a larger set of 
```

Comment:
When I read something like this, I say "reference?".

You might just say that it's been proven to be "a" great way.